tests/_support/AcceptanceTester.php

```diff
@@ -68,6 +69,39 @@ public function generateEvent() {
 		return array_merge( self::$defaultEvent, $random );
 	}
 
+	/**
+	 * Update Event
+	 */
+	public function edit_event( $updateEvent ) {
+		$originalTitle		= $updateEvent['originalTitle'];
+		$newTitle			= $updateEvent['newTitle'];
+		$content			= $updateEvent['content'];
+		$allDay 			= $updateEvent['allDay'];
+		if(empty($updateEventp['originalTitle']) || empty($updateEvent['newTitle'])){
+            throw new \InvalidArgumentException('Title missing');
+        }
```

Comment:
Hahaha...that looks the same, but it isn't :) The first block uses spaces before `throw new` and the last brace whereas the second block uses tabs.